apps/ui/src/App.vue

```diff
@@ -75,7 +75,8 @@ watch(route, () => {
 });
 
 watch(isSwiping, () => {
-  if (String(route.name).startsWith('site-')) return;
+  if (String(route.name).startsWith('site-') || window.innerWidth > 1012)
```

Comment:
I wonder if it makes sense to import it from Tailwind config to make sure those are in sync:
```js
const lgWidth = Number(tailwindConfig.theme.screens.lg.replace('px', ''));
```